src/Agent.Listener/LocalRunner.cs

```diff
@@ -79,6 +94,36 @@ public sealed override void Initialize(IHostContext hostContext)
                 return Constants.Agent.ReturnCode.Success;
             }
 
+            // Verify the current directory is the root of a git repo.
+            string repoDirectory = Directory.GetCurrentDirectory();
+            if (!Directory.Exists(Path.Combine(repoDirectory, ".git")))
+            {
+                throw new Exception("Unable to run the build locally. The command must be executed from the root directory of a local git repository.");
+            }
+
+            // Get the URL - required if missing tasks.
+            string url = command.GetUrl(suppressPrompt: true);
```

Comment:
Oops suppressPrompt=true means the url doesn't get run through the validator... I think I need to call it again with suppressPrompt=false (when not empty) to make sure the validator is run. I'm basically working around a limitation in the command line parser that I don't want to spend time fixing right now.